### DISCUSSION TITLE
230807 주간 스프린트 회의 (17:00 ~ )

### DISCUSSION THREAD

**Discussion by seokhwan-an** [woowacourse-teams/2023-shook#217] [회의]
스프린트 회의를 진행한다.

**Comment by splitCoding** [comment]
---

## 메인페이지


### 킬링파트 등록 이벤트 섹션

- 새로 추가할 노래에 대해서 등록기간을 정한다.
- 유저들이 자신의 킬링파트를 지정해서 등록한다.

### 노래 목록 조회 섹션

- 좋아요가 많은 노래 순서대로 조회



---

## 노래 상세 페이지


###  킬링파트 듣기

- 킬링파트를 들을 수 있다.
- 킬링파트 3개를 좋아요가 높은 순으로 정렬하여 표시한다.

### 킬링파트 좋아요, 댓글 기능

- 킬링파트에 좋아요를 누를 수 있다.
    - 좋아요를 한번더 누르면 취소된다.
    - 계정당 1번만 누를 수 있다.

- 킬링파트에 댓글을 남길 수 있다.
	- 계정당 1번만 남길 수 있다.

### 이전, 다음 노래 이동 기능

- 유저가 다시 목록으로 가야하는 불편함을 해소하기 위해
